hudi-client/src/main/java/org/apache/hudi/config/HoodieStorageConfig.java

```diff
@@ -81,6 +87,7 @@ public Builder fromProperties(Properties props) {
 
     public Builder limitFileSize(long maxFileSize) {
       props.setProperty(PARQUET_FILE_MAX_BYTES, String.valueOf(maxFileSize));
+      props.setProperty(HFILE_FILE_MAX_BYTES, String.valueOf(maxFileSize));
```

Comment:
@vinothchandar I see that you split this function into file specific functions. That doable but with more base file formats added, it may be cumbersome and verbose to keep adding the .limitXXXFileSize for specific formats. 

I am ok either way.